docs: readme update (#15)
kawalcovid19/silacak: README.md

> # SILACAK

Reviewer: ```suggestion
# Silacak
```

Nitpick: Silacak (sentence case) instead of all uppercase, as dictated by the new design.

A progressive web app (PWA) to track COVID19 cases.

## Purpose and Principles

> Help field officer (TNI) to easily tracing the cases. According to WHO Tracing ratio should be 1:15.

Reviewer: ```suggestion
The main purpose of Silacak is to help our field officers (TNI) easily trace COVID-19 cases in Indonesia. According to WHO Tracing ratio should be 1:15.
```

- Grammar.
- For the second sentence, we need to provide links to any official source to further drive the purpose of this project.

### Principles

- Working without reliable connection (even offline mode)
- New user can input the data easily.
- Performance

> - Secure

Reviewer: For this, we also need to further elaborate on these points, just like what we did on [wargabantuwarga.com](https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/README.md#principles)

We can also copy those principles over here. What do you think, @zainfathoni?

Reviewer 2: Agreed. However, copying over the content potentially causes them to be out-of-sync when we're updating one of them.
Therefore, please add a link to [the **Principles** section at the original document](https://github.com/kawalcovid19/wargabantuwarga.com#principles) instead.

## Developer

Setting Up Silacak in Local.

This is an installation guide for developers before start the project:
Prerequisite:

1.

> 

Reviewer: ```suggestion
```

## Getting Started

To get started running the project locally, please follow the steps below.

First, clone the repository:

```bash
git clone https://github.com/kawalcovid19/silacak.git
```

Then, install dependencies and fetch data to your local machine:

```bash

> cd wargabantuwarga.com

Reviewer: ```suggestion
cd silacak
```

yarn install

> yarn mirror-box

Reviewer: ```suggestion
```

This command is not necessary in Silacak.

> ```

Reviewer 3: Is this part relevant to this project? 🤔

Author: I am not sure for this one

Reviewer 3: I think this part is relevant to our other project, the wargabantuwarga.com

Reviewer 2: Exactly, it's irrelevant here. We're neither using the `mirror-box` data nor exporting this site statically.

Finally, run the development server:

```bash
yarn dev
```

Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.

## Contributing

See our contribution guidelines [here](contributing.md)

> ## Authors and acknowledgment
> 

Reviewer: ```suggestion
```

> Thanks goes to these wonderful people
> 

Reviewer: ```suggestion
```

> ## License
> 

Reviewer: ```suggestion
```
